packages/satcheljs-stitch/lib/stitch.ts

```diff
@@ -27,6 +27,16 @@ export function subscribe<T extends ActionHandler>(actionType: string, callback:
     handlers[actionType].push(callback);
 }
 
+export function unsubscribe<T extends ActionHandler>(actionType: string, callback: T) {
+    const actionHandlers = handlers[actionType] || [];
```

Comment:
why not return an unsubscribe function that closures the index from subscribe()? that's more idiomatic JS IMO.